server/src/test/java/org/elasticsearch/transport/CompressibleBytesOutputStreamTests.java

```diff
@@ -1,114 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0 and the Server Side Public License, v 1; you may not use this file except
- * in compliance with, at your election, the Elastic License 2.0 or the Server
- * Side Public License, v 1.
- */
-
-package org.elasticsearch.transport;
-
-import org.elasticsearch.common.bytes.BytesReference;
-import org.elasticsearch.common.compress.CompressorFactory;
-import org.elasticsearch.common.io.stream.BytesStream;
-import org.elasticsearch.common.io.stream.BytesStreamOutput;
-import org.elasticsearch.common.io.stream.InputStreamStreamInput;
-import org.elasticsearch.common.io.stream.StreamInput;
-import org.elasticsearch.test.ESTestCase;
-
-import java.io.EOFException;
-import java.io.IOException;
-
-public class CompressibleBytesOutputStreamTests extends ESTestCase {
```

Comment:
For the record, `InboundDecoderTests#testCompressedDecode` and friends are enough to cover this code now. We `rarely()` enable compression in internal cluster tests but apparently never in real cluster tests, maybe we should?

Reply:
I once wanted to do this in the REST tests (~2 years ago) and was told we're already randomizing too much :) Maybe it's actually unnecessary since there is not practical difference between real and internal cluster tests here anyway. I'll leave it as is because of past discussions on randomizing more network settings in real cluster tests but I think I'd also be +0 on this. Maybe we can revisit this some other time :)